### PR TITLE
Unarchive Instrument Data 

### PR DESCRIPTION
**Pull request by jasonwalker80**
intital revision of model command to unarchive instrument data.

### CODE REVIEW

File: lib/perl/Genome/Model/Command/UnarchiveInstrumentData.pm [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                }
+            }
+        }
+
+        $self->unarchive_additional_data($instrument_data);
+    }
+}
+
+sub valid_statuses {
+    return ('In Progress', 'Hold', 'Pending');
```
Comment (tmooney) [inline lib/perl/Genome/Model/Command/UnarchiveInstrumentData.pm]: This subroutine doesn't really make sense for models.

File: lib/perl/Genome/Model/Command/UnarchiveInstrumentData.pm [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+sub unarchive_additional_data {
+    #this is a hook for overriding--by default do nothing;
+    return 1;
```
Comment (tmooney) [inline lib/perl/Genome/Model/Command/UnarchiveInstrumentData.pm]: To be useful this probably needs the the `Site::TGI` addition like the AnP version.  The two commands might benefit from a common base (an InstrumentData unarchive command?) that they can both use.

**Comment by kkrysiak** [comment]
I don't feel qualified to confirm the code on this pull request but I would definitely benefit from its intended utility.

**Comment by jasonwalker80** [comment]
CI, test this

**Comment by jasonwalker80** [comment]
CI, test this

**Comment by jasonwalker80** [comment]
CI, test this

**Pull request merged**